### DISCUSSION TITLE
Question about units

### DISCUSSION THREAD

**Discussion by vgenisel** [micom-dev/micom#128] [Q&A]
Hi, maybe a silly question but just to confirm:
When I import diets from the VMH database the fluxes are in mmol/day.
If I follow the workflow for designing new mediums (using the VMH template available on micom-dev/media) and then use the new medium to predict growth rates and production rates in MICOM are the units still in a daily scale? (1/day and mmol/gDW*day, respectively).
Or somehow, they were converted to 1/h and mmol/gDW*h?
Many thanks

**Comment by cdiener** [comment]
That is true. You would usually have to divide by the length of the steady state (in h) and adjust for bacterial biomass. However, that never worked well for me since the VMH diets only cover a few metabolites, so growth rates are already pretty low. So I usually use the initial amounts without making them even smaller and then complete the medium (which adds fluxes in the units of `mmol/(gDW*h)`). So the media are still assumed to be in units of mmol/(gDW * h) overall.

**Reply by cdiener** [reply]
There is also a [related discussion](https://github.com/micom-dev/micom/discussions/115).

**Comment by vgenisel** [comment]
Hi. Thanks for the explanation, that was very useful.
I compared the influence of imported VMH fluxes (modified or not to mmol/day) and indeed, making them smaller increased the numerical instability. Using the fluxes in their original unit produces better results.
